src/libraries/System.IO.FileSystem/tests/Directory/CreateTempSubdirectory.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Directory_CreateTempSubdirectory : FileSystemTest
```

Comment:
Maybe verify that the filemode of the created directory is `700` on Unix. That is an important trait of this API.

Reply:
Q: is it on Windows also guaranteed the resulting directory is only accessible to the current user?

Reply:
> Maybe verify that the filemode of the created directory is 700 on Unix.

Done.

> Q: is it on Windows also guaranteed the resulting directory is only accessible to the current user?

No, on Windows there is no guarantee on the permissions of the directory. Just that it is empty, and it is in the TEMP directory.